src/components/calcite-action/calcite-action.scss

```diff
@@ -139,3 +139,6 @@
   right: unset;
   left: var(--calcite-app-side-spacing-half);
 }
+
+:host([loading]) {
```

Comment:
nitpick: some of these components won't need any host styling, they would just show a loading spinner inside. Maybe we should remove the host styling here.